src/plugins/copyPaste/focusableElement.js

```diff
@@ -170,10 +172,13 @@ function destroyElement(wrapper) {
     refCounter = 0;
 
     // Detach secondary element from the DOM.
+    const secondaryElement = secondaryElements.get(wrapper.rootDocument);
+
     if (secondaryElement && secondaryElement.parentNode) {
       secondaryElement.parentNode.removeChild(secondaryElement);
-      secondaryElement = null;
+      secondaryElements.set(wrapper.rootDocument, null);
```

Comment:
I think that [Map.prototype.delete()](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Map/delete) / [WeakMap.prototype.delete()
](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap/delete) should be used here.